[PATCH v6] Speed up make check

JannePeltonen opened the pull request:
Make time validation test and timer_accuracy example complete faster to speed up make check

JannePeltonen commented on test/validation/api/time/time.c:
```diff
@@ -405,20 +407,20 @@ static void time_test_wait_until(time_cb time_cur, time_from_ns_cb time_from_ns)
 	odp_time_t lower_limit, upper_limit;
 	odp_time_t start_time, end_time, wait;
-	odp_time_t second = time_from_ns(ODP_TIME_SEC_IN_NS);
+	odp_time_t second = time_from_ns(WAIT_TIME);
```
Renamed in v2, also in an another function.

JannePeltonen commented:
v4: Added missing EXTRA_DIST for the added shell script.

psavol commented on test/validation/api/time/time.c:
```diff
@@ -543,54 +564,16 @@ static void time_test_accuracy_nsec(time_nsec_cb time_nsec)
+		/* Check that ODP time is within +-5% of system time */
+		CU_ASSERT(sec_t < sec_c * 1.05);
+		CU_ASSERT(sec_t > sec_c * 0.95);
+	}
```
When one test fails with a loop, it's hard to tell from a log which one failed. Maybe unroll the loop, or print out something on a failure. Also the loop on line 520 could be changed the same way.

JannePeltonen commented:
v5:
- added the requested printouts to identify failed tests better
- added a new patch to remove a redundant part of time monotony test to make it run faster

TuomasTaipale reviewed (approved):
One commit-message related comment, otherwise looks OK to me.

JannePeltonen commented:
v6: Fixed the typo and added reviewed-by tags.
